packages/backend/server/src/plugins/copilot/context/resolver.ts

```diff
@@ -103,6 +103,9 @@ class CopilotContextDoc implements ContextDoc {
 
   @Field(() => SafeIntResolver)
   createdAt!: number;
+
```

Comment:
should rebase `darksky/improve-context-api`

Reply:
Only need the type in this PR.